plugins/tech-insights/api-report.md

```diff
@@ -87,6 +115,11 @@ export class TechInsightsClient implements TechInsightsApi {
   // (undocumented)
   getCheckResultRenderers(types: string[]): CheckResultRenderer[];
   // (undocumented)
+  getLatestFacts(
```

Comment:
what constitutes "latest"? i guess the backend decides right

Reply:
I believe that whoever is calling needs to have a reference that are the updated records that will be returned in this method, does it make sense?

I think a simple getFacts makes it very generic to whoever is consuming the api